pilot/pkg/networking/core/v1alpha3/gateway.go

```diff
@@ -44,6 +44,17 @@ import (
 	"istio.io/pkg/log"
 )
 
+// DisableGatewayPortTranslationLabel is a label on Service that declares that, for that particular
```

Comment:
Does it have to be label ? ( i.e. searching for it or using the endpoint label ?)

Reply:
It could be an annotation, but our internal model.Service doesn't keep track of annotations and I didn't want a big change. Probably not the best reason, but this is a workaround already...

Is the concern just label vs annotation? Or some completely different way to signal this?

Reply:
Since it's experimental - ok to use label.